widgets/Couchbase-batchsource.json

```diff
@@ -0,0 +1,143 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Couchbase",
+  "configuration-groups": [
+    {
+      "label": "General",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "csv",
+          "label": "Nodes",
+          "name": "nodes"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Bucket",
+          "name": "bucket"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Select Fields",
+          "name": "selectFields",
+          "widget-attributes": {
+            "default": "*"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Conditions",
+          "name": "conditions"
+        }
+      ]
+    },
+    {
+      "label": "Credentials",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Username",
+          "name": "username"
+        },
+        {
+          "widget-type": "password",
+          "label": "Password",
+          "name": "password"
+        }
+      ]
+    },
+    {
+      "label": "Error Handling",
```

Comment:
nit: move this property into the 'Advanced' section instead of having a section with just one property.

Reply:
Done.